onnx/backend/test/case/node/split.py

```diff
@@ -374,4 +382,276 @@ def export_2d_uneven_split_opset18() -> None:
+        node = onnx.helper.make_node(
+            "Split",
+            inputs=["input"],
+            outputs=["output_1", "output_2", "output_3", "output_4"],
+            num_outputs=4,
+        )
+
+        expected_outputs = [
+            np.array([1.0, 2.0]).astype(np.float32),
+            np.array([3.0, 4.0]).astype(np.float32),
+            np.array([5.0, 6.0]).astype(np.float32),
+            np.array([7.0]).astype(np.float32),
+        ]
+        expect(
+            node,
+            inputs=[node_input],
+            outputs=expected_outputs,
+            name="test_split_1d_uneven_split_opset20",
+        )
+
+    @staticmethod
+    def export_2d_uneven_split_1_opset20() -> None:
+        node_input = np.array(
+            [
+                [1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0],
+                [9.0, 10.0, 11.0, 12.0, 13.0, 14.0, 15.0, 16.0],
+            ]
+        ).astype(np.float32)
+
+        node = onnx.helper.make_node(
+            "Split",
+            inputs=["input"],
+            outputs=["output_1", "output_2", "output_3"],
+            axis=1,
+            num_outputs=3,
+        )
+
+        expected_outputs = [
+            np.array([[1.0, 2.0, 3.0], [9.0, 10.0, 11.0]]).astype(np.float32),
+            np.array([[4.0, 5.0, 6.0], [12.0, 13.0, 14.0]]).astype(np.float32),
+            np.array([[7.0, 8.0], [15.0, 16.0]]).astype(np.float32),
+        ]
+
+        expect(
+            node,
+            inputs=[node_input],
+            outputs=expected_outputs,
+            name="test_split_2d_uneven_split_opset20",
+        )
+
+    @staticmethod
+    def export_2d_uneven_split_2_opset20() -> None:
+        node_input = np.array(
+            [
+                [1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0],
+                [8.0, 9.0, 10.0, 11.0, 12.0, 13.0, 14.0],
+            ]
+        ).astype(np.float32)
+
+        node = onnx.helper.make_node(
+            "Split",
+            inputs=["input"],
+            outputs=["output_1", "output_2", "output_3"],
+            axis=1,
+            num_outputs=3,
+        )
+
+        expected_outputs = [
+            np.array([[1.0, 2.0, 3.0], [8.0, 9.0, 1.0]]).astype(np.float32),
+            np.array([[4.0, 5.0], [11.0, 12.0]]).astype(np.float32),
+            np.array([[6.0, 7.0], [13.0, 14.0]]).astype(np.float32),
+        ]
+
+        expect(
+            node,
+            inputs=[node_input],
+            outputs=expected_outputs,
+            name="test_split_2d_uneven_split_2_opset20",
+        )
+
+    @staticmethod
+    def export_2d_uneven_split_3_opset20() -> None:
+        node_input = np.array(
+            [
+                [1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0],
+                [8.0, 9.0, 10.0, 11.0, 12.0, 13.0, 14.0],
+            ]
+        ).astype(np.float32)
+
+        node = onnx.helper.make_node(
+            "Split",
+            inputs=["input"],
+            outputs=["output_1", "output_2", "output_3"],
+            axis=0,
```

Comment:
Instead of changing the `axis`, it would make sense to change the new attribute introduced `minimize_diff` or whatever it is called. At least, I am looking for some test-case to show the difference that attribute makes, whether that is this test or some other test.